javascript/javascript1/week2/student-manager.js

```diff
@@ -0,0 +1,48 @@
+//Student manager
+const class07Students = [];
+function addStudentToClass(studentName)
+{
+    if(studentName === "")
+    {
+        console.log("please provide student name");
+    }
+    else if(studentName === "Queen Margrethe")
+    {
+        class07Students.push(studentName);
+        console.log(studentName + " has been added to class" )
+    }
+    else if(class07Students.length < 6)
+    {
+            if(class07Students.includes(studentName))
```

Comment:
does this need to be nested?

Reply:
I tryed to fix it or I can do something else.